packages/waggle/src/storage/storage.ts

```diff
@@ -366,6 +366,24 @@ export class Storage {
     }
   }
 
+  public async uploadFile(file: FileContent) {
+    // Create directory for file
+    await this.ipfs.files.mkdir(`/${file.dir}`, { parents: true })
+    // Write file to IPFS
+    const uploadingFileName = `${file.name}.${file.ext}`
```

Comment:
Maybe we should add some kind of uuid to the filename? User can send many images with the same name.